ui/web/index.html

```diff
@@ -3,10 +3,316 @@
+                        <option value="L">L</option>            
+                    </select>
+                </td>
+                </td>
+            </tr>
+        </table>
+    </div> 
+
+    </fieldset>
+
+    <fieldset>
+
+        <legend>Load</legend>
+
+        <div id="place">
+
+            <input type="submit" name="but" id="loadTasksButton" value="Load">
+
+        </div>
+
+    </fieldset>
+    <fieldset>
+
+        <legend>Print</legend>
+
+        <div id="schedule">
+
+            <table border="0" widtd="100%">
+                <tr>
+                    <td>Tasks
+                        <select name="tasks" id="tasks">
+                            <option value=""></option>
+                            <option value="0">Task 1</option>
+                            <option value="1">Task 2</option>
+                            <option value="2">Task 3</option>
+                            <option value="3">Task 4</option>
+                            <option value="4">Task 5</option>
+                            <option value="5">Task 6</option>
+                            <option value="6">Task 7</option>
+                            <option value="7">Task 8</option>
+                            <option value="8">Task 9</option>
+                            <option value="9">Task 10</option>
+                            <option value="10">Task 11</option>
+                            <option value="11">Task 12</option>            
+                        </select>                       
+                    </td>
+                    <td>
+                        <input type="button" value="Print" id="loadPrint">
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                        <div id="chartdiv" style="height:500px;width:300px; "></div>
+                    </td>
+                </tr>
+                </table>
+    </fieldset>
+
+    <script>  
+
+        var applicationContext = {};
+
+        var tasksObject = {};
+        var tasksPlot = {
+            tasksObject
+        };
+                                   
+        function loadTasks() {
+            var columns_mapping = {};
+            var requestBody = {
+            columns_mapping
+            };
+            var excelFile = $('#excelFile').get(0).files.item(0);
+            requestBody.sheet = $("#sheetName").val();
+            requestBody.first_row = +$("#firstRow").val();
+            requestBody.last_row = +$("#lastRow").val();
+            requestBody.rows_to_skip = $("#rowSkip").val();
+            columns_mapping.uid = $("#uid").val();
+            columns_mapping.name = $("#nam").val();
+            columns_mapping.blockers = $("#blockers").val();
+            columns_mapping.min_estimate = $("#minEst").val();
+            columns_mapping.normal_estimate = $("#norEst").val();
+            columns_mapping.max_estimate = $("#maxEst").val(); 
```

Comment:
Let's rewrite `requestBody` creation in more short way:

```
var requestBody = {
  sheet: $("#sheetName").val(),
  first_row: +$("#firstRow").val(),
  last_row: +$("#lastRow").val(),
  rows_to_skip: $("#rowSkip").val(),
  columns_mapping: {
    uid: $("#uid").val(),
    name: $("#nam").val(),
    blockers: $("#blockers").val(),
    min_estimate: $("#minEst").val(),
    normal_estimate: $("#norEst").val(),
    max_estimate: $("#maxEst").val()
  }
}
```

and move it more close to its usage, it means below to success function of http://localhost:5000/api/excelWorkbook call.

Reply:
requestBody rewritten